assets/src/application/services/toaster/withAnimation.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useMemo, useRef, useState } from 'react';
+import { useTransition, animated } from 'react-spring';
+
+import getAnimation from './utilities/getAnimation';
+import getToastContainerStyle from './utilities/getToastContainerStyle';
+import { PositionsType, ToastProps, WithAnimationProps } from './types';
+import useTimeout from '../utilities/useTimeout';
+
+/**
+ * adapted from:
+ * @link https://github.com/bmcmahen/toasted-notes/blob/master/src/Alert/Message.ts
```

Comment:
https://github.com/bmcmahen/toasted-notes/blob/master/src/Alert/Message.tsx

Reply:
how did i manage to mess this up for every single link?